selfservice/flow/registration/hook.go

```diff
@@ -169,15 +168,10 @@ func (e *HookExecutor) PostRegistrationHook(w http.ResponseWriter, r *http.Reque
 		WithRequest(r).
 		WithField("identity_id", i.ID).
 		Info("A new identity has registered using self-service registration.")
-	trace.SpanFromContext(r.Context()).AddEvent(
-		semconv.EventIdentityCreated,
-		trace.WithAttributes(
-			attribute.String(semconv.AttrIdentityID, i.ID.String()),
-			attribute.String(semconv.AttrNID, i.NID.String()),
-			attribute.String(semconv.AttrClientIP, httpx.ClientIP(r)),
-			attribute.String("flow", string(a.Type)),
-		),
-	)
+
+	events.Emit(r.Context(), events.IdentityCreated,
+		semconv.AttrIdentityID(i.ID),
+		events.AttrFlowType(a.Type))
 
 	s, err := session.NewActiveSession(r, i, e.d.Config(), time.Now().UTC(), ct, identity.AuthenticatorAssuranceLevel1)
```

Comment:
Should we emit a `SessionIssued` Event here?

Reply:
I think so, but actually in:
https://github.com/ory/kratos/blob/master/selfservice/hook/session_issuer.go, I think.

The session is created, but only returned if the session hook is set.

Reply:
I believe all places where `SessionIssued` is required are already instrumented.

Reply:
Added an event to the session issuer hook

Reply:
Actually I might have lied. There are more instances where we issue sessions. For example during recovery flows. Those don't currently have audit logs either.

Could be covered in https://github.com/ory/kratos/issues/3241